packages/lfo-input/src/LFOInput.ts

```diff
@@ -0,0 +1,188 @@
+import {
+  getNextEnumValue,
+  handleEachInput,
+  HedronEngine,
+  InputOptionNodesConfig,
+  IPlugin,
+  NodeValue,
+} from '@hedron/engine'
+
+const TAU = Math.PI * 2
+const lerp = (v0: number, v1: number, t: number) => (1 - t) * v0 + t * v1
+
+export class LFOInput implements IPlugin {
+  public readonly id = 'lfo-input'
+  public readonly name = 'LFO Input'
+  public readonly inputType = 'lfo'
+  public readonly description =
+    'Generates LFO waves (e.g. sin, square, sawtooth) as inputs for params.'
+  public readonly optionNodesConfig = [
```

Comment:
Input plugins might want option nodes. This config is the same format as the sketch params config. These nodes will be generated for any new input created

Reply:
Love this pattern, would be cool if it's also the case for top-level settings of any node, like those for audio smoothing/falloff, etc